src/extension/options.js

```diff
@@ -37,7 +37,7 @@ function isValidGitHubURL(giturl) {
 }
 
 function isValidHost(host) {
-  const hostPattern = /^$|^([a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}$/;
+  const hostPattern = /^$|^([a-zA-Z0-9-._]*\.)*([a-zA-Z0-9]+\.)+[a-zA-Z]+$/;
```

Comment:
The last-but-second segment can also contain fashes or underscores, e.g. `www.example-dash.com`
```suggestion
  const hostPattern = /^$|^([a-zA-Z0-9-._]*\.)*([a-zA-Z0-9-_]+\.)+[a-zA-Z]+$/;
```

Reply:
with the above regex, below example hostnames are tested:

- example.com
- sub.eample.com
- sub-dash.example.com
- sub.sub2-dash.example.com
- sub.example-dash.com
- sub123.example.com
- sub.sub123.example.com
- www.thisislongerthansixtyonecharacterswhichisnotallowedbythestandards.com